test/browser/browser.sh

```diff
@@ -49,25 +49,28 @@ for retry in $(seq 5); do
     sleep $((5 * retry * retry))
 done
 
+CONTAINER="$(cat .cockpit-ci/container)"
+
+# import the test CONTAINER image as a directory tree for nspawn
+mkdir /var/tmp/tasks
+podman export "$(podman create --name tasks-import $CONTAINER)" | tar -x -C /var/tmp/tasks
```

Comment:
You might want to look into `podman mount`...

Reply:
Well no -- *not* having created podman containers and storage files was the whole point of that exercise..